### PR TITLE
Created GroundController and handlers

### LINKED ISSUE #8: Create utils.py for Common Methods
Includes error_dict() and success_dict() and replacing current uses

### LINKED ISSUE #12: Create CommandManager
Create Class to handle socket communication (command sending) to Server 1
Create Command class which has routing information intended for Pixhawk

Notes:

Instantiated by app.py 

Accesses:
- TelemetryController (access latest telemetry to update routing and flight plan)
- AllQR (access QR information to create and update flight plan)
- Waypoints and Routes

Features:
- Creates, stores and updates flight plan
- Updates flight plan according to boundary
- Has a current Command and Route
- Uses recent telemetry to know when close to endpoint
         - Sends next Command to server to CS-Flight when close
- Will process return ComandOK messages from Server 1
         - /verify-command
         - Verifies command received is command sent

### LINKED ISSUE #15: Create LoggingHandler
Similar to Logging class used by past repos
Need to import logging, set up directory to handle logs
Add logs to gitignore

### LINKED ISSUE #16: Create GroundController
GroundController class which is highest level class, and will call sub level classes for processing

Also responsible to setting up all the sub classes/managers/controllers.

Example
/Set-Telemetry
- Calls TelemetryController to update telemetry on site and data held
- Calls BoundaryManager to check if boundaries crossed
- Calls CommandManager to check if command almost finished and send new command
- Uses LoggingManager

### PR DESCRIPTION
**Pull request by CraigP17**
Closes #8 
Closes #12 
Closes #15 
Closes #16

### CODE REVIEW

File: server/commandManager.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    def connect_to_socket(self):
+        """Start socket connection
+
+        :return: True if successful connection, else False
+        """
+        return self.socket.connect()
+
+    def send_command(self, message):
+        """Send message to Flight
```
Comment (JacobRajah) [inline server/commandManager.py:47]: Would this be to the communication router instead?

File: server/obsolete/async_socket.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import asyncore
+import logging
+import socket
+from time import sleep
+
+
+class FlaskClientSocketASYNC(asyncore.dispatcher_with_send):
```
Comment (JacobRajah) [inline server/obsolete/async_socket.py:7]: Just curious, the plans for this class?

**Review by JacobRajah — APPROVED** [review]

File: server/obsolete/async_socket.py (hunk 2 above)
Comment (CraigP17) [reply server/obsolete/async_socket.py:7]: Was exploring other socket modules that have async functionality so nothing is blocked while reading/sending.
Currently not used, but may be used for CommunicationRouter if that's happening

Kept code in /obsolete folder to store not currently used scripts and scratch files

File: server/commandManager.py (hunk 1 above)
Comment (CraigP17) [reply server/commandManager.py:47]: Technically yes, with intended target Flight

**Review by JacobRajah — APPROVED** [review]

**Pull request merged**